### PR TITLE
[Themes 2012] Inline Editing | Small Promo Block

### PR DESCRIPTION
**Pull request by malavikakoppula**
#### Jira Ticket: [THEMES-2012](https://arcpublishing.atlassian.net/browse/THEMES-2012)

inline edit certain content elements on a small promo block within PageBuilder Editor, so that I can curate the homepage small promo to drive engagement.

## Test Steps

1. Checkout this branch `git checkout THEMES-2012`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmwdia/small-promo-block`
3. _id: LF33TMKJENGHFOSIKPHHIHFHJM
4. publish the page and edit the heading

Before:
Screenshot 2024-08-22 at 4 22 23 PM

After:
Screenshot 2024-08-22 at 4 22 47 PM

[THEMES-2012]: https://arcpublishing.atlassian.net/browse/THEMES-2012?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: blocks/small-promo-block/features/small-promo/default.jsx [hunk 1]
```diff
@@ -35,8 +35,9 @@ const SmallPromo = ({ customFields }) => {
 	} = customFields;
 	const { registerSuccessEvent } = useComponentContext();
 	const { arcSite, isAdmin } = useFusionContext();
-	const { searchableField } = useEditableContent();
+	const { editableContent, searchableField } = useEditableContent();
 	const shouldLazyLoad = lazyLoad && !isAdmin;
+	const editable = true;
```
Comment (nschubach) [inline blocks/small-promo-block/features/small-promo/default.jsx]: editable will always be true, so it's not needed...

**Review by nschubach — CHANGES_REQUESTED** [review]
There's either an unnecessary flag here or missing logic.

File: blocks/small-promo-block/features/small-promo/default.jsx (hunk 1 above)
Comment (malavikakoppula) [reply blocks/small-promo-block/features/small-promo/default.jsx]: @nschubach  Removed editable

**Comment by blakeganderson** [comment]
Updated PR without package-lock changes [here](https://github.com/WPMedia/arc-themes-blocks/pull/2248)

**Pull request closed without merging**